.gitignore

```diff
@@ -11,3 +11,4 @@ format.bat
 .install
 .lua/*
 .vscode
+CHANGELOG.MD
```

Comment:
👀 what did you test there? 😄

Reply:
i was playing around with the bigwigs packager and i did find a way to get it to install externals into a working copy, but not to keep it from generating changelogs